scripts/add_extra_components.py

```diff
@@ -222,6 +223,128 @@ def attach_stores(n, costs, config):
         )
 
 
+def attach_cooking_technologies(n, cooking_costs, config):
```

Comment:
- not sure what happening here in detail... 
- would like to see a passing CI first
- then see some images if the behaviour is working as in the minimal example